daisy_workflows/image_build/centos7.wf.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Name": "build-rhelcent67",
```

Comment:
build-centos-7

I'm thinking we should drop project, zone and gcspath from here and document that those need to be passed in as flags or added to a forked workflow. Or put them in as vars (the flag overrides will override vars).